Packs/CarbonBlackProtect/Integrations/CarbonBlackProtect/CarbonBlackProtect.py

```diff
@@ -544,26 +544,26 @@ def update_computer(id, name, computer_tag, description, policy_id, automatic_po
     :param template: True if computer is a template
     :return: Result json of the request
     """
-    body_params = {
-        'id': id,
-        'name': name,
-        'computerTag': computer_tag,
-        'description': description,
-        'policyId': policy_id,
-        'automaticPolicy': automatic_policy,
-        'localApproval': local_approval,
-        'refreshFlags': refresh_flags,
-        'prioritized': prioritized,
-        'debugLevel': debug_level,
-        'kernelDebugLevel': kernel_debug_level,
-        'debugFlags': debug_flags,
-        'debugDuration': debug_duration,
-        'cCLevel': cclevel,
-        'cCFlags': ccflags,
-        'forceUpgrade': force_upgrade,
-        'template': template,
-    }
-    body_params = remove_keys_with_empty_value(body_params)
+    raw_computer = get_computer(id)
```

Comment:
Way do you need the raw_computer and not directly assign body_params to get_computer(id) ? @dansterenson

Reply:
Fixed